Loop54.Shared/Model/Request/GetPopularOrRecentEntitiesRequest.cs

```diff
@@ -43,8 +51,8 @@ protected GetPopularOrRecentEntitiesRequest(string behaviorType, string[] entity
         public string[] EntityType { get; set; }
 
         /// <summary>
-        /// A user ID (normally the same as the one in the User-Id header) to retrieve the most common/recent entities for that user or null to
-        /// retrieve the globally most common/recent entities.
+        /// A user ID to retrieve the most common/recent entities for that user or null to retrieve the globally most common/recent entities.
+        /// The literal "(CurrentUser)" can be specified to use the user ID from the User-Id request header.
```

Comment:
Good idea to add this info here! Shouldn't we encourage callers to use the new constant, though?

Reply:
I changed it according to the API documentation. 


> Shouldn't we encourage callers to use the new constant, though?

But it makes sense. How about giving a reference to this constant like I did now?

Reply:
That's good!